File: web/js/video.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */


function crearVideo(cliente) {
    var jsonParam = {'video': true, 'cliente': cliente};
    $.ajax({
        async: false,
        type: 'POST',
        url: 'Logindos',
        data: jsonParam,
        success: function (data) {
            console.log("termino")
//             alert(JSON.stringify(data));
                 var nombre=data.imagen[0].nombre;
                 console.log(nombre);
             
            var div=$('#contenedor_video');
            $('.video_contenedor').remove();
            div.append('<video class="video_contenedor" src="http://200.75.13.14:8082/Capturas/Videos/'+nombre+'" controls></video>');
        },
        error: function (jqXHR, textStatus, errorThrown) {
            console.log("Error");
            console.log(jqXHR.toString());
            console.log(textStatus.toString());
            console.log(errorThrown.toString());
        }
    });



}

File: src/java/Model/domain/Video.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Model.domain;

/**
 *
 * @author Sebastian Oliveros
 */
public class Video {
    public int id_video;
    public String nombre;
    public String fecha;
    public int id_usuario;

    public int getId_video() {
        return id_video;
    }

    public void setId_video(int id_video) {
        this.id_video = id_video;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getFecha() {
        return fecha;
    }

    public void setFecha(String fecha) {
        this.fecha = fecha;
    }

    public int getId_usuario() {
        return id_usuario;
    }

    public void setId_usuario(int id_usuario) {
        this.id_usuario = id_usuario;
    }
    
}
